### PR TITLE
Add Server Side Config Option For VOIP Host Address

### PR DESCRIPTION
**Pull request by mjwells2002**
Addresses #228 by adding a server side config option for VOIP Host Address

### CODE REVIEW

File: src/main/java/de/maxhenkel/voicechat/voice/client/ClientVoiceEvents.java [hunk 1]
```diff
@@ -82,10 +82,15 @@ public void authenticate(UUID playerUUID, InitPacket initPacket) {
         ClientPacketListener connection = minecraft.getConnection();
         if (connection != null) {
             try {
+                String ip;
                 SocketAddress socketAddress = ((IClientConnection) connection.getConnection()).getChannel().remoteAddress();
                 if (socketAddress instanceof InetSocketAddress) {
                     InetSocketAddress address = (InetSocketAddress) socketAddress;
-                    String ip = address.getHostString();
+                    ip = address.getHostString();
+                    String _initIP = initPacket.getVoiceHost();
```
Comment (henkelmax) [inline src/main/java/de/maxhenkel/voicechat/voice/client/ClientVoiceEvents.java]: No underscores

File: src/main/java/de/maxhenkel/voicechat/voice/client/ClientVoiceEvents.java [hunk 2]
```diff
@@ -82,10 +82,15 @@ public void authenticate(UUID playerUUID, InitPacket initPacket) {
...
+                    if (_initIP != "") {
```
Comment (henkelmax) [inline src/main/java/de/maxhenkel/voicechat/voice/client/ClientVoiceEvents.java]: String.isEmpty() is the better solution for this.

**Review by henkelmax — CHANGES_REQUESTED** [review]

**Comment by mjwells2002** [comment]
changes done

**Review by henkelmax — CHANGES_REQUESTED** [review]
You somehow edited `gradlew`

**Comment by mjwells2002** [comment]
ah yes that is it being marked with the +x flag to make it executable on Linux as gradlew itself is just a shell script and it gets extremely annoying to use if it is not executable annoyingly git shows this as an empty change to file when it is just a permission change

**Pull request merged**